src/scripts/crank.ts

```diff
@@ -83,19 +84,39 @@ setInterval(
 
 
 async function run() {
-  const spotMarkets = await Promise.all(
-    markets[cluster].map((m) => {
-      return Market.load(
-        connection,
-        new PublicKey(m.address),
-        {
-          skipPreflight: true,
-          commitment: 'processed' as Commitment,
-        },
-        serumProgramId,
-      );
-    }),
-  );
+  let spotMarkets;
+  if (TOP_MARKET === 'false' || TOP_MARKET === undefined) {
```

Comment:
btw you can just say `if (!TOP_MARKET)`, or even put the next if-branch first and do`if (TOP_MARKET) {//fetch} else {// static}`

just cosmetic tho, doesn't matter

Reply:
if I put just ``` if (!TOP_MARKET)``` then someone will put false in the env variable will also go through because env is treated like a string so that's why I wrote it like this